docs/i18n.stories.mdx

```diff
@@ -0,0 +1,52 @@
+<Meta title="Documentation/i18n" />
+
+# i18n
+
+## Default
+
+By default `carbon` uses `en-GB` translations. **No action required**.
+
+Preview [src/locales/en-gb.js](https://github.com/Sage/carbon/blob/master/src/locales/en-gb.js)
+
+## Resources
+
+To add a new locale and use it with `carbon`, create a new locale file based on [interface](https://github.com/Sage/carbon/blob/master/src/components/i18n-provider/index.d.ts) or [proptypes](https://github.com/Sage/carbon/blob/master/src/components/i18n-provider/index.js) of `I18nProvider` component.
```

Comment:
Same with the links here. Redirected to an `error 404` page.

Reply:
Will be available there when pull request is merged.